<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1051</width>
    <height>620</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QSplitter" name="splitter2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>1051</width>
      <height>551</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QSplitter" name="splitter1">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QWidget" name="verticalLayoutWidget_3">
      <layout class="QVBoxLayout" name="verticalLayout1">
       <item>
        <widget class="QTreeWidget" name="groupWidget">
         <property name="font">
          <font>
           <pointsize>9</pointsize>
          </font>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
         <property name="rootIsDecorated">
          <bool>false</bool>
         </property>
         <column>
          <property name="text">
           <string notr="true">Name</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget_4">
      <layout class="QVBoxLayout" name="verticalLayout2">
       <item>
        <widget class="QTreeWidget" name="windowsGroupWidget">
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
         <property name="rootIsDecorated">
          <bool>false</bool>
         </property>
         <column>
          <property name="text">
           <string notr="true">Windows groups</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget_5">
     <layout class="QVBoxLayout" name="verticalLayout3">
      <item>
       <widget class="QTreeWidget" name="entryWidget">
        <property name="font">
         <font>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="rootIsDecorated">
         <bool>false</bool>
        </property>
        <column>
         <property name="text">
          <string notr="true">Title</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>User name</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Password</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>URL</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Notes</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1051</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSaveAs"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionAddGroup"/>
    <addaction name="actionEditGroup"/>
    <addaction name="actionRemoveGroup"/>
    <addaction name="separator"/>
    <addaction name="actionAddEntry"/>
    <addaction name="actionEditEntry"/>
    <addaction name="actionRemoveEntry"/>
    <addaction name="separator"/>
    <addaction name="actionAdd_windows_group"/>
    <addaction name="actionEdit_windows_group"/>
    <addaction name="actionRemove_windows_group"/>
    <addaction name="separator"/>
    <addaction name="actionAdd_windows_user"/>
    <addaction name="actionEdit_windows_user"/>
    <addaction name="actionRemove_windows_user"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
   <addaction name="separator"/>
   <addaction name="actionAddGroup"/>
   <addaction name="actionRemoveGroup"/>
   <addaction name="actionEditGroup"/>
   <addaction name="separator"/>
   <addaction name="actionAddEntry"/>
   <addaction name="actionEditEntry"/>
   <addaction name="actionRemoveEntry"/>
   <addaction name="separator"/>
   <addaction name="actionAdd_windows_group"/>
   <addaction name="actionEdit_windows_group"/>
   <addaction name="actionRemove_windows_group"/>
   <addaction name="separator"/>
   <addaction name="actionAdd_windows_user"/>
   <addaction name="actionEdit_windows_user"/>
   <addaction name="actionRemove_windows_user"/>
  </widget>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="../resources/iconpaths.qrc">
     <normaloff>:/icons/open.ico</normaloff>:/icons/open.ico</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="../resources/iconpaths.qrc">
     <normaloff>:/icons/save.ico</normaloff>:/icons/save.ico</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="text">
    <string>Save as...</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionAddGroup">
   <property name="icon">
    <iconset resource="../resources/iconpaths.qrc">
     <normaloff>:/icons/addGroup.ico</normaloff>:/icons/addGroup.ico</iconset>
   </property>
   <property name="text">
    <string>Add group</string>
   </property>
  </action>
  <action name="actionEditGroup">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../resources/iconpaths.qrc">
     <normaloff>:/icons/editGroup.png</normaloff>:/icons/editGroup.png</iconset>
   </property>
   <property name="text">
    <string>Edit group</string>
   </property>
  </action>
  <action name="actionRemoveGroup">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../resources/iconpaths.qrc">
     <normaloff>:/icons/removeGroup.ico</normaloff>:/icons/removeGroup.ico</iconset>
   </property>
   <property name="text">
    <string>Remove group</string>
   </property>
  </action>
  <action name="actionAddEntry">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../resources/iconpaths.qrc">
     <normaloff>:/icons/addEntry.ico</normaloff>:/icons/addEntry.ico</iconset>
   </property>
   <property name="text">
    <string>Add entry</string>
   </property>
  </action>
  <action name="actionEditEntry">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../resources/iconpaths.qrc">
     <normaloff>:/icons/editEntry.png</normaloff>:/icons/editEntry.png</iconset>
   </property>
   <property name="text">
    <string>Edit entry</string>
   </property>
  </action>
  <action name="actionRemoveEntry">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../resources/iconpaths.qrc">
     <normaloff>:/icons/removeEntry.ico</normaloff>:/icons/removeEntry.ico</iconset>
   </property>
   <property name="text">
    <string>Delete entry</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="../resources/iconpaths.qrc">
     <normaloff>:/icons/new.png</normaloff>:/icons/new.png</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionAdd_windows_group">
   <property name="icon">
    <iconset resource="../resources/iconpaths.qrc">
     <normaloff>:/icons/addWinGroup.png</normaloff>:/icons/addWinGroup.png</iconset>
   </property>
   <property name="text">
    <string>Add windows group</string>
   </property>
  </action>
  <action name="actionEdit_windows_group">
   <property name="icon">
    <iconset resource="../resources/iconpaths.qrc">
     <normaloff>:/icons/editWinGroup.png</normaloff>:/icons/editWinGroup.png</iconset>
   </property>
   <property name="text">
    <string>Edit windows group</string>
   </property>
  </action>
  <action name="actionRemove_windows_group">
   <property name="icon">
    <iconset resource="../resources/iconpaths.qrc">
     <normaloff>:/icons/removeWinGroup.png</normaloff>:/icons/removeWinGroup.png</iconset>
   </property>
   <property name="text">
    <string>Remove windows group</string>
   </property>
  </action>
  <action name="actionAdd_windows_user">
   <property name="icon">
    <iconset resource="../resources/iconpaths.qrc">
     <normaloff>:/icons/addWinUser.png</normaloff>:/icons/addWinUser.png</iconset>
   </property>
   <property name="text">
    <string>Add windows user</string>
   </property>
  </action>
  <action name="actionEdit_windows_user">
   <property name="icon">
    <iconset resource="../resources/iconpaths.qrc">
     <normaloff>:/icons/editWinUser.png</normaloff>:/icons/editWinUser.png</iconset>
   </property>
   <property name="text">
    <string>Edit windows user</string>
   </property>
  </action>
  <action name="actionRemove_windows_user">
   <property name="icon">
    <iconset resource="../resources/iconpaths.qrc">
     <normaloff>:/icons/removeWinUser.png</normaloff>:/icons/removeWinUser.png</iconset>
   </property>
   <property name="text">
    <string>Remove windows user</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../resources/iconpaths.qrc"/>
 </resources>
 <connections/>
</ui>
